tequilapi/endpoints/identities.go

```diff
@@ -101,6 +106,22 @@ func (endpoint *identitiesApi) Register(resp http.ResponseWriter, request *http.
 	resp.WriteHeader(http.StatusAccepted)
 }
 
+func (endpoint *identitiesApi) Unlock(resp http.ResponseWriter, request *http.Request, params httprouter.Params) {
+	id := params.ByName("id")
+	unlockReq, err := toUnlockRequest(request)
+	if err != nil {
+		utils.SendError(resp, err, http.StatusBadRequest)
```

Comment:
Should not validation errors be sent with `utils.SendValidationErrorMessage()`?

Reply:
In most of endpoints, request parsing happens in two stages:
- Mapping of json into go struct,
- Validating go struct - whether required data is present and in correct format.

If #1 stage fails and json is not parsed into our struct, BadRequest is returned.
If #1 succeeds, than #2 follow, which might return validation errors.

This error we're discussing is due to #1 - failing to parse json.

Reply:
1st stage 400 - got it
2nd stage - so I missed this stage, validation of DTO fields

Reply:
There is only one field - password, which is optional. What validation are you missing?